com.xycz.simple_live.yaml

```diff
@@ -0,0 +1,279 @@
+        config-opts:
+          - -Dlibmpv=true
+          - -Dbuild-date=false
+          - -Dalsa=disabled
+          - -Dlua=disabled
+          - -Dcplayer=false
+        sources:
+          - type: archive
+            url: https://github.com/mpv-player/mpv/archive/v0.39.0.tar.gz
+            sha256: 2ca92437affb62c2b559b4419ea4785c70d023590500e8a52e95ea3ab4554683
+            x-checker-data:
+              type: anitya
+              project-id: 5348
+              stable-only: true
+              url-template: https://github.com/mpv-player/mpv/archive/v$version.tar.gz
+              is-important: true
+
+        modules:
+
+          - name: libv4l2
+            buildsystem: meson
+            cleanup:
+              - /sbin
+              - /bin
+              - /include
+              - /lib/pkgconfig
+              - /share/man
+              - /share/doc
+            config-opts:
+              - -Dbpf=disabled
+              - -Dudevdir=/app/lib/udev
+              - -Dsystemdsystemunitdir=/dev/null
+            sources:
+              - type: archive
+                url: https://linuxtv.org/downloads/v4l-utils/v4l-utils-1.28.1.tar.xz
+                sha256: 0fa075ce59b6618847af6ea191b6155565ccaa44de0504581ddfed795a328a82
+                x-checker-data:
+                  type: anitya
+                  project-id: 9998
+                  stable-only: true
+                  url-template: https://linuxtv.org/downloads/v4l-utils/v4l-utils-$version.tar.xz
+
+          - name: nv-codec-headers
+            cleanup:
+              - "*"
+            no-autogen: true
+            make-install-args:
+              - PREFIX=/app
+            sources:
+              - type: git
+                url: https://git.videolan.org/git/ffmpeg/nv-codec-headers.git
+                tag: n12.2.72.0
+                commit: c69278340ab1d5559c7d7bf0edf615dc33ddbba7
+                x-checker-data:
+                  type: git
+                  tag-pattern: ^n([\d.]+)$
+
+          - name: libopenmpt
+            config-opts:
+              - --disable-static
+              - --disable-tests
+              - --disable-examples
+              - --disable-openmpt123
+              - --disable-doxygen-doc
+              - --without-portaudio
+            cleanup:
+              - /include
+              - /lib/pkgconfig
+              - /lib/*.la
+              - /share/doc
+            sources:
+              - type: archive
+                url: https://lib.openmpt.org/files/libopenmpt/src/libopenmpt-0.7.10+release.autotools.tar.gz
+                sha256: 093713c1c1024f4f10c4779a66ceb2af51fb7c908a9e99feb892d04019220ba1
+                x-checker-data:
+                  type: anitya
+                  project-id: 141468
+                  stable-only: true
+                  url-template: https://lib.openmpt.org/files/libopenmpt/src/libopenmpt-$version+release.autotools.tar.gz
+
+          - name: ffmpeg
+            cleanup:
+              - /include
```

Comment:
why not use the ffmpeg extension in runtime?

Reply:
Replaced with ffmpeg-full extension.